autora_closed_loop_demo/researcher_hub/autora_workflow.py

```diff
@@ -229,18 +230,63 @@ def trial_list_to_experiment_data(trial_sequence):
     independent: coherence_ratio, motion_direction
     dependent: d_prime
     """
+    trial_sequence = pd.read_csv("../../myexperiment.csv").fillna(pd.NA)
+    
+    trial_sequence["trial_number"] = np.nan
+    index = 1
+    counter = 1
+    for i in range(26, len(trial_sequence) - 2):
+        if counter > 20:
+            counter = 1
+            index = index +1
+        trial_sequence.loc[i, "trial_number"] = index
+        counter = counter + 1
+        print(counter, "counter")
+        
+    # print(trial_sequence.loc[41, "bean_choices"])
+    # print(trial_sequence.isnull())
+    # exit()
+    # trial_sequence.replace(r'^\s*$', pd.NA, regex=True)
+    trial_sequence =  trial_sequence.ffill() # translate coherent_ration and movement_direction to the input row. 
+    trial_sequence.to_csv("vi.csv") 
+    trial_sequence= trial_sequence[(trial_sequence['bean_choices'] == '["1","2","3","4","x"]') & (trial_sequence["trial_number"] > 0)] # remove all other rows
+    trial_sequence = trial_sequence[['coherence_movement','coherent_movement_direction', 'trial_number', 'response', 'bean_correct_key']] # remove unnecessary columns
+    trial_sequence = trial_sequence.groupby(['trial_number', 'bean_correct_key', 'coherence_movement', 'coherent_movement_direction']).agg(lambda x: tuple(x)) 
+    trial_sequence['hit'] = pd.NA
+    trial_sequence['miss'] = pd.NA
+    for trial in trial_sequence: 
```

Comment:
@MoAljam, Can you please do the comparison between these two columns and add the value to the hits and misses.